### PR TITLE
Fixes #1491 by updating help message for rpc call z_importkey

### LINKED ISSUE #1491: z_importkey help message displays a label argument
z_importkey takes two arguments: `zkey` and `rescan` but the help message displays a `label` argument that is not processed.

```
 if (fHelp || params.size() < 1 || params.size() >  3)
        throw runtime_error(
            "z_importkey \"zkey\" ( \"label\" rescan )\n"
...
            "\nExport a zkey\n"
            + HelpExampleCli("z_exportkey", "\"myaddress\"") +
            "\nImport the zkey with rescan\n"
            + HelpExampleCli("z_importkey", "\"mykey\"") +
            "\nImport using a label and without rescan\n"
            + HelpExampleCli("z_importkey", "\"mykey\" \"testing\" false") +
            "\nAs a JSON-RPC call\n"
            + HelpExampleRpc("z_importkey", "\"mykey\", \"testing\", false")
        );
```

### PR DESCRIPTION
**Pull request by bitcartel**

### CODE REVIEW

File: src/wallet/rpcdump.cpp [hunk 1]
```diff
@@ -534,7 +534,7 @@ Value z_importkey(const Array& params, bool fHelp)
 
     if (fHelp || params.size() < 1 || params.size() > 3)
```
Comment (coinspect) [inline src/wallet/rpcdump.cpp]: `params.size() > 2`

Comment (bitcartel) [inline src/wallet/rpcdump.cpp]: Thanks, fixed now.

**Comment by daira** [comment]
ACK. Do we have tests that passing more than the specified number of arguments to any of the new RPC calls will fail?

**Comment by bitcartel** [comment]
@daira There were tests for some of the calls.  New commit adds tests for those not covered.

File: src/wallet/rpcwallet.cpp [hunk 2]
```diff
@@ -2792,7 +2792,7 @@ Value z_getnewaddress(const Array& params, bool fHelp)
     if (!EnsureWalletIsAvailable(fHelp))
         return Value::null;
 
-    if (fHelp || params.size() > 1)
+    if (fHelp || params.size() > 0)
```
Comment (daira) [inline src/wallet/rpcwallet.cpp]: I thought ``getnewaddress ""`` was intended to be allowed?

Comment (bitcartel) [inline src/wallet/rpcwallet.cpp]: This is for `z_getnewaddress` not `getnewaddress`.

**Comment by daira** [comment]
ACK. @zkbot r+

**Pull request merged**